script/release-workflow/docker-prepare.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -euo >/dev/null
+
+docker run --privileged --rm tonistiigi/binfmt --install all
+docker buildx create --platform linux/arm64,linux/arm/v8,linux/amd64 --name multi_arch_builder
+docker buildx use multi_arch_builder
```

Comment:
sets up a multi-arch builder so `amd64` machines can build for `arm` and `arm64` with `qemu`